src/main/java/racingcar/domain/Cars.java

```diff
@@ -2,58 +2,47 @@
 
 import java.util.LinkedList;
 import java.util.List;
-import racingcar.dto.CarDto;
+import java.util.stream.Collectors;
 import racingcar.util.IntGenerator;
+import racingcar.util.RacingCarIntGenerator;
 
 public class Cars {
 
-    List<Car> cars;
+    private final List<Car> cars;
 
     public Cars(List<String> carNames) {
-        this.cars = new LinkedList<>();
-        carNames.forEach(carName -> cars.add(new Car(carName)));
+        this(carNames, new RacingCarIntGenerator());
     }
 
     public Cars(List<String> carNames, IntGenerator intGenerator) {
         this.cars = new LinkedList<>();
         carNames.forEach(carName -> cars.add(new Car(carName, intGenerator)));
     }
 
-    public Cars(List<Car> cars, boolean isTest) { // TODO : 인자 없애는 법 고민
+    public Cars(List<Car> cars, boolean isTest) {
         this.cars = cars;
     }
 
     public void requestMoveEachCar() {
         cars.forEach(Car::tryMove);
     }
 
-    public List<CarDto> getStatuses() {
-        List<CarDto> carDtos = new LinkedList<>();
-        cars.forEach(car -> carDtos.add(car.getStatus()));
-        return carDtos;
-    }
-    
-    public List<CarDto> getWinner() {
-        List<CarDto> carStatuses = this.getStatuses();
-        int winnerCount = 0;
-        List<CarDto> winnerStatuses = new LinkedList<>();
-        for (CarDto carStatus : carStatuses) {
-            winnerCount = getMaxMovedCount(winnerCount, carStatus.getMovedCount());
-        }
-        for (CarDto carStatus : carStatuses) {
-            appendIfHaveSameMovedCount(winnerStatuses, carStatus, winnerCount);
-        }
-        return winnerStatuses;
-    }
-    
-    private int getMaxMovedCount(int winnerCount, int movedCount) {
-        return Math.max(winnerCount, movedCount);
-    }
-    
-    private void appendIfHaveSameMovedCount(List<CarDto> winners, CarDto carDto, int winnerCount) {
-        if (carDto.getMovedCount() == winnerCount) {
-            winners.add(carDto);
-        }
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    public List<Car> getWinner() {
```

Comment:
```suggestion
    public List<Car> getWinners() {
```
복수형 응답을 메서드명으로 표시해주면 좋을 것 같네요 😄

Reply:
그렇네요 ..! 놓쳤던 부분이네요 감사합니다!